### ISSUE TITLE
nvtrust/host_tools/python/gpu_cc_tool.py “xxx” not defined, e.g. NvSwitch

### ISSUE THREAD

**Issue by JustPlay** [NVIDIA/nvtrust#14]
```
root@p133-011-144:~/tdx/scripts/nvidia.d# python3 nvtrust/host_tools/python/gpu_cc_tool.py 
NVIDIA GPU Tools version 535.86.06
  File "nvtrust/host_tools/python/gpu_cc_tool.py", line 125, in find_gpus_sysfs
    dev = NvSwitch(dev_path=dev_path)
2023-10-02,21:47:12.955 ERROR    GPU /sys/bus/pci/devices/0000:03:00.0 broken: name 'NvSwitch' is not defined
Traceback (most recent call last):
  File "nvtrust/host_tools/python/gpu_cc_tool.py", line 125, in find_gpus_sysfs
    dev = NvSwitch(dev_path=dev_path)
NameError: name 'NvSwitch' is not defined

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "nvtrust/host_tools/python/gpu_cc_tool.py", line 2480, in <module>
    main()
  File "nvtrust/host_tools/python/gpu_cc_tool.py", line 2424, in main
    gpus, other = find_gpus()
  File "nvtrust/host_tools/python/gpu_cc_tool.py", line 146, in find_gpus
    return find_gpus_sysfs(bdf)
  File "nvtrust/host_tools/python/gpu_cc_tool.py", line 137, in find_gpus_sysfs
    dev = BrokenGpu(dev_path=dev_path)
  File "nvtrust/host_tools/python/gpu_cc_tool.py", line 1262, in __init__
    super(BrokenGpu, self).__init__(dev_path)
  File "nvtrust/host_tools/python/gpu_cc_tool.py", line 1092, in __init__
    self.parent = PciDevice.find_or_init(sysfs_find_parent(dev_path))
  File "nvtrust/host_tools/python/gpu_cc_tool.py", line 1040, in find_or_init
    dev = PciDevice.init_dispatch(dev_path)
  File "nvtrust/host_tools/python/gpu_cc_tool.py", line 1026, in init_dispatch
    cls = PciDevice.find_class_for_device(dev_path)
  File "nvtrust/host_tools/python/gpu_cc_tool.py", line 992, in find_class_for_device
    pci_dev = PciDevice(dev_path)
  File "nvtrust/host_tools/python/gpu_cc_tool.py", line 1092, in __init__
    self.parent = PciDevice.find_or_init(sysfs_find_parent(dev_path))
  File "nvtrust/host_tools/python/gpu_cc_tool.py", line 1040, in find_or_init
    dev = PciDevice.init_dispatch(dev_path)
  File "nvtrust/host_tools/python/gpu_cc_tool.py", line 1026, in init_dispatch
    cls = PciDevice.find_class_for_device(dev_path)
  File "nvtrust/host_tools/python/gpu_cc_tool.py", line 992, in find_class_for_device
    pci_dev = PciDevice(dev_path)
  File "nvtrust/host_tools/python/gpu_cc_tool.py", line 1092, in __init__
    self.parent = PciDevice.find_or_init(sysfs_find_parent(dev_path))
  File "nvtrust/host_tools/python/gpu_cc_tool.py", line 1040, in find_or_init
    dev = PciDevice.init_dispatch(dev_path)
  File "nvtrust/host_tools/python/gpu_cc_tool.py", line 1026, in init_dispatch
    cls = PciDevice.find_class_for_device(dev_path)
  File "nvtrust/host_tools/python/gpu_cc_tool.py", line 997, in find_class_for_device
    return IntelRootPort
NameError: name 'IntelRootPort' is not defined
```

**Comment by steven-bellock** [comment]
Presumably this is fixed by #11.

**Comment by JustPlay** [comment]
Fixed in https://github.com/NVIDIA/nvtrust/commit/6296b7a8dce48a65202fcf7358453874d47ed6a0
